k8s/packer/testground-ami.json

```diff
@@ -0,0 +1,24 @@
+  {
+  "variables": {
+    "aws_access_key": "",
+    "aws_secret_key": "",
+    "aws_region": "",
+    "source_ami": ""
+  },
+  "builders": [{
+    "type": "amazon-ebs",
+    "access_key": "{{user `aws_access_key`}}",
+    "secret_key": "{{user `aws_secret_key`}}",
+    "region": "{{user `aws_region`}}",
+    "source_ami": "{{user `source_ami`}}",
+    "instance_type": "m4.xlarge",
```

Comment:
I don't think it must be the same. This is just the VM used to build the AMI - it could be even a smaller instance, although it doesn't really matter much, as this instance is up for 1-2min.